src/deepsparse/image_classification/README.md

```diff
@@ -0,0 +1,244 @@
+```pycon
+from sparsezoo import Zoo
+
+# you can lookup an appropriate model stub here: https://sparsezoo.neuralmagic.com/
+model_stub = "zoo:cv/classification/resnet_v1-50/pytorch/sparseml/imagenet/pruned95-none"
+
+# directly download the model data to your local directory
+model = Zoo.download_model_from_stub(model_stub)
+
+# the onnx model file is there, ready for deployment
+import os 
+os.path.isfile(os.path.join(model.dir_path, "model.onnx"))
+>>>True
+```
+
+
+## Deployment APIs
+
+[DeepSparse] provides both a python Pipeline API and an out-of-the-box model 
+server that can be used for end-to-end inference in either existing python 
+workflows or as an HTTP endpoint. Both options provide similar specifications 
+for configurations and support a variety of Image Classification models.
+
+### Python API
+
+Pipelines are the default interface for running the inference with the 
+[DeepSparse] Engine.
+
+Once a model is obtained, either through [SparseML] training or directly from [SparseZoo],
+`deepsparse.Pipeline` can be used to easily facilitate end to end inference and deployment
+of the sparsified image classification model.
+
+If no model is specified to the `Pipeline` for a given task, the `Pipeline` will automatically
+select a pruned and quantized model for the task from the `SparseZoo` that can be used for accelerated
+inference. Note that other models in the [SparseZoo] will have different tradeoffs between speed, size,
+and accuracy.
+
+To learn about sparsification in more detail, refer to [SparseML docs](https://docs.neuralmagic.com/sparseml/)
+
+### HTTP Server
+
+As an alternative to Python API, the DeepSparse inference server allows you to 
+serve ONNX models and pipelines in HTTP. Both configuring and making requests 
+to the server follow the same parameters and schemas as the Pipelines enabling 
+simple deployment. Once launched, a `/docs` endpoint is created with full 
+endpoint descriptions and support for making sample requests.
+
+Example deployment using a 95% pruned resnet50 is given below
+For full documentation on deploying sparse image classification models with the
+DeepSparse Server, see the [documentation](https://github.com/neuralmagic/deepsparse/tree/main/src/deepsparse/server).
+
+##### Installation
+
+The deepsparse server requirements can be installed by specifying the `server` 
+extra dependency when installing DeepSparse.
+
+```bash
+pip install deepsparse[server]
+```
+
+## Deployment Use Cases
+
+The following section includes example usage of the Pipeline and server APIs for
+various image classification models. 
+
+[List of Image Classification SparseZoo Models](https://sparsezoo.neuralmagic.com/?domain=cv&sub_domain=classification&page=1)
+
+
+#### Python Pipeline
+
+```python
+from deepsparse import pipeline
+cv_pipeline = pipeline(
+  task='image_classification', 
+  model_path='zoo:cv/classification/resnet_v1-50/pytorch/sparseml/imagenet/pruned95-none',  # Path to checkpoint or SparseZoo stub
+)
```

Comment:
```suggestion
```python
from deepsparse import Pipeline
cv_pipeline = Pipeline.create(
  task='image_classification', 
  model_path='zoo:cv/classification/resnet_v1-50/pytorch/sparseml/imagenet/pruned95-none',  # Path to checkpoint or SparseZoo stub
)
```